doc/man/8/mount.ceph.rst

```diff
@@ -184,8 +184,11 @@ Mount only part of the namespace/file system::
     mount.ceph :/some/directory/in/cephfs /mnt/mycephfs
 
 Mount non-default FS, in case cluster has multiple FSs::
-
-    mount -t ceph :/ /mnt/mycephfs2 -o mds_namespace=mycephfs2
+    mount -t ceph :/ /mnt/mycephfs2 -o fs=mycephfs2
+    
+    or
+    
+    mount -t ceph :/ /mnt/mycephfs2 -o mds_namespace=mycephfs2 # This option name is deprecated and will be removed in a future release.
```

Comment:
In practice, I don't see us ever removing this option completely. We can encourage users to use the new syntax, but we will need to carry the old one for a long time. There is little to be gained by removing it entirely.